apis/slo/v1alpha1/pod.go

```diff
@@ -64,3 +64,19 @@ func GetPodMemoryQoSConfig(pod *corev1.Pod) (*PodMemoryQOSConfig, error) {
 	}
 	return &cfg, nil
 }
+
+const (
+	// AnnotationPodEnableCoreSched is the annotation key of the Linux Core Scheduling enabled.
+	// Valued "true" means the core sched is enabled. Valued "false" means the core sched is disabled.
+	// Valued other values or missing the annotation are ignored and the core sched status will be kept.
+	AnnotationPodEnableCoreSched = apiext.DomainPrefix + "enable-core-sched"
+
+	// AnnotationCoreSchedGroupID is the annotation key of the group ID of the Core Sched.
+	// Core Sched: https://docs.kernel.org/admin-guide/hw-vuln/core-scheduling.html
+	// Pods with the different core sched group IDs will not be running at the same SMT core at the same time, which
+	// means they will take different core sched cookies. And a pod with cookie is allowed to run on the same core with
+	// a pod who does not set any cookie (cookie=0).
+	// NOTE: If a pod sets the core sched enabled but misses the core sched ID, the pod will be assigned an individual
+	// and stateless internal ID according to the pod's UID.
+	AnnotationCoreSchedGroupID = apiext.NodeDomainPrefix + "/core-sched-group-id"
```

Comment:
Is there any format or specification requirement for the value corresponding to this annotation?

Reply:
added